Asking for Pull Req

roniy68 opened the pull request.

roniy68 commented:
Asking For Code Review

BregornOriginal commented on index.html:
```diff
@@ -7,7 +7,6 @@
 <body>
 
```
- [ ] As you were working with [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md), kindly follow that flow according to the rules. This Pull Request should be from a feature to the main branch. 💯 
- [ ] As Linters are really useful to avoid mistakes before doing a Pull Request or merging a branch, they are mandatory for the whole program on Microverse. Please make sure that your linters are working when you create a Pull Request. [Here](https://github.com/microverseinc/linters-config) You can follow the instructions to install them (HTML and CSS only for now) 🥇 
- [ ] Kindly when you create a Pull Request, it's better to add a description about what are you implementing on this PR. You can follow the requirements as a description on it.

roniy68 replied:
I have deleted all the comments... 
sorry I forgot about the messy ones. Can you please review the code once again ?

roniy68 replied:
And also about the pull request that I didnot give a brief description.

BregornOriginal reviewed (changes requested):
## Hi @roniy68  

## Good job so far!
## There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ README looks professional

## Required Changes ♻️
_Check the comments under the review._

![one-step-closer-one-more](https://user-images.githubusercontent.com/93630700/204548165-3c84e5cd-bfb3-4088-8ba1-b200c6dff8dd.gif)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

christianbiring1 commented on README.md:
```diff
@@ -1,27 +1,4 @@
 <a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
```
- [x] Kindly make sure you use the correct GitHub flow.
![image](https://user-images.githubusercontent.com/97100091/204556408-9535d499-a181-4b50-aae2-cbc17634cdf9.png)
The Pull request is supposed to be `into main form feature_branch` and not the other way around
- [x] Also consider giving your PR a descriptive title telling what the project is about.
`hint: Hello-microverse` sound better than `asking for pull request` :face_with_spiral_eyes:

abdulhamiid replied:
As requested by the previous reviewer, please kindly use correct Github flow in your preject: 
MEANING: You are required to create a pull request from `feature branch` to the `main branch`. 
`feature branch` is a branch you have worked on with the new features.
 
Learn more about [Correct Github flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)

christianbiring1 commented on README.md:
```diff
@@ -1,27 +1,4 @@
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
```
- [ ] Kindly make sure your project has a descriptive summary telling in detail what changes were made.
![image](https://user-images.githubusercontent.com/97100091/204557767-c94e6e06-2cb5-40e8-a749-e5d3a4897cc5.png)

You may consider clicking those three dots on the right and from there you will be able to edit. :smile:

christianbiring1 commented on README.md:
```diff
@@ -1,27 +1,4 @@
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
```
- [ ] Kindly make sure your linters are st correctly and no one fails to pass the test.
`Hint: You may consider reading the html-css linters documentation` [here](https://github.com/microverseinc/linters-config/tree/master/html-css#:~:text=README.md-,HTML%20%26%20CSS3%20Course,-If%20you%20are)

abdulhamiid replied:
- [ ] As requested by the previous reviewers, please kindly consider adding code linters correctly for this repository, **Code linters** help to diagnose and fix technical issues—e.g. code smells—in the code.
Please kindly visit the html-css in the [Linters Config](https://github.com/microverseinc/linters-config/tree/master/html-css) 😊 and follow the steps closely.

- [ ] 🟢 Check linters:
Linter result should be green ✔️ . AKA: Linter should report NO errors.
There should be no custom changes to linter config (as an attempt to cheat linter error reports).

- [ ] If you need help on how to setup linters correctly, kindly ask for help on the slack Chanel **"microverse full-time stundent"** and you will surely get someone to help. 😊

christianbiring1 commented on index.html:
```diff
@@ -7,7 +7,6 @@
 <body>
 
 <h1>Hello Microverse</h1>
-<p>hello Microvese from Ahmed Hasan Rony</p>
```
- [ ] As said in the project requirement, kindly make sure you add some style to your created element thanks its class in css

roniy68 replied:
I already added some css but with the heading tag but not with the class. Now I have added the class.

christianbiring1 reviewed (changes requested):
Hi @roniy68 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# STATUS: REQUEST CHANGES :recycle: 
![almost](https://user-images.githubusercontent.com/97100091/204558958-4ad69731-ae7f-431b-8aee-d72fe5407e2a.gif)


 ## Highlights

 ## Required Changes ♻️
- [ ] Kindly make sure you have an A test.md file with the text "This file should be ignored by git" and it must be ignored by git.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

roniy68 commented:
I tried to correct all the Requirements my code reviewers asked. I have added CSS the right way with a class name to the h1 tag.
I have corrected the blank spaces and Added some more content.

abdulhamiid reviewed (changes requested):
# Status: Required Changes :recycle:
Hi @roniy68 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._



### Your pull request summary 👇 
![image](https://user-images.githubusercontent.com/78251927/204613466-809d0670-8b15-4c8b-b1d5-e283e3524a0e.png)

As requested by the previous reviewer, please kindly note that pull requests should have a summary describing the introduced changes in more detail.
- [ ] Example 👇 
``` 
 In this project, I
- Added HTML and do some styling
- Setting up linters
- Added .gitignore file to my project
```

You can learn more about [Professional basic repo rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
